p2p/discover/table.go

```diff
@@ -44,6 +44,10 @@ const (
 
 	maxBondingPingPongs = 16
 	maxFindnodeFailures = 5
+
+	autoRefreshInterval = 1 * time.Hour
+	seedCount           = 30
+	seedMaxAge          = 5 * 24 * time.Hour
```

Comment:
Do we need this max age thing? Nodes older than 24h should be automatically evacuated if the node has a healthy connection, so given that you were recently connected, everything should be fresh. On the other hand if you weren't connected for a long time, then at least give a fighting chance with the seeds and not simply disable them altogether. I'd rely on the expiration mechanism for cleanup and not limit the max are during seed retrieval.